plugin/src/main/java/net/dzikoysk/funnyguilds/telemetry/metrics/MetricsCollector.java

```diff
@@ -37,36 +41,43 @@ public void start() {
 
     @Override
     public void run() {
-        // mcstats
+        // MCStats
         MCStats mcstats = this.mcstats;
         if (mcstats != null) {
             MCStats.Graph global = mcstats.createGraph("Guilds and Users");
+
             global.addPlotter(new MCStats.Plotter("Guilds") {
                 @Override
                 public int getValue() {
                     return GuildUtils.getGuilds().size();
                 }
             });
+
             global.addPlotter(new MCStats.Plotter("Users") {
                 @Override
                 public int getValue() {
                     return UserUtils.usersSize();
```

Comment:
tutaj można dać `plugin.getUserManager().usersSize()`

Reply:
W ogóle ta metoda to się powinna nazywać `countUsers()`